run.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
+export UNICAST_HOSTS_FILE=${UNICAST_HOSTS_FILE:=/hosts/unicast_hosts.txt}
+
+[ -f $UNICAST_HOSTS_FILE ] && cp $UNICAST_HOSTS_FILE /elasticsearch/config/unicast_hosts.txt
```

Comment:
Should we use headless service for nodes discovery as suggested in https://github.com/fabric8io/elasticsearch-cloud-kubernetes#kubernetes-cloud-plugin-for-elasticsearch?